packages/manager/modules/emailpro/src/dashboard/emailpro.controller.js

```diff
@@ -512,5 +518,9 @@ export default /* @ngInject */ function EmailProCtrl(
       searchText: $scope.exchange.domain,
     });
 
+  $scope.isMigratedMxPlan = () => {
+    return $scope.exchange?.isMXPlan && $scope?.isMxPlanMigrated;
+  };
```

Comment:
```suggestion
  $scope.isMigratedMxPlan = $scope.exchange?.isMXPlan && $scope.isMxPlanMigrated;
```
Doesn't make sense because `$scope` can't be null and you can use a property rather a function as it never changes

Reply:
was a mistake